### PR TITLE
Make plugin compatible with JOSM >= 13733

### LINKED ISSUE #17: Problem with the newest josm version 13878
Opening mbtiles file generates unexpected error. Previous version of josm has no issues.

### PR DESCRIPTION
**Pull request by floscher**
* add badges to README // 894718d431fa2bff3aa9bce3f202fccfce9d7f89
* update gradle-josm-plugin to latest version 0.4.5 and Gradle to 4.7 // d2da1d6565bc1a94a88c98da1c650bebe4b08956
* update to use newer JOSM API >= 13733 // 926650a2d617d20a114d9b6c942ab0d1bd208e11

Fixes #17.

### CODE REVIEW

File: src/org/openstreetmap/josm/plugins/mbtiles/MbtilesTileLoaderFactory.java [hunk 1]
```diff
@@ -16,8 +16,7 @@ public MbtilesTileLoaderFactory(Connection connection) {
     }
 
     @Override
-    public TileLoader makeTileLoader(TileLoaderListener listener, Map<String, String> headers) {
+    public TileLoader makeTileLoader(TileLoaderListener listener, Map<String, String> headers, long minimumExpiryTime) {
```
Comment (floscher) [inline src/org/openstreetmap/josm/plugins/mbtiles/MbtilesTileLoaderFactory.java]: @iandees the new parameter `minimumExpiryTime` is currently ignored, optimally it should be respected in the tile loader factory.

Comment (iandees) [reply src/org/openstreetmap/josm/plugins/mbtiles/MbtilesTileLoaderFactory.java]: Do you know what it represents?

Comment (floscher) [reply src/org/openstreetmap/josm/plugins/mbtiles/MbtilesTileLoaderFactory.java]: As far as I understand, it represents the minimum amount of time for which tiles should remain in the cache (even if the server says the expiry time should be shorter).
See https://josm.openstreetmap.de/ticket/16249 for the development discussion about this change.

**Comment by don-vip** [comment]
Plugin needs a new update after [14632](https://josm.openstreetmap.de/changeset/14632/josm)

**Pull request merged**